core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.directive.html

```diff
@@ -153,8 +153,7 @@
          aria-expanded="false"
          aria-label="User Menu"
          tabindex="0"
-         ng-mouseover="$ctrl.openSubmenu($event, 'profileMenu')"
-         ng-mouseleave="$ctrl.closeSubmenuIfNotMobile($event)">
+         ng-mouseover="$ctrl.openSubmenu($event, 'profileMenu')">
```

Comment:
This changes the functionality right? Now it would stay open even after moving the mouse away? I don't think that is expected...

Reply:
Okay, I'll try to fix this in another way. Thanks!